modules/aspect_bazel_lib/0.11.0/MODULE.bazel

```
@@ -16,4 +12,6 @@ bazel_dep(name = "stardoc", version = "0.5.0", repo_name = "io_bazel_stardoc")
 
 ext = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "ext")
 use_repo(ext, "jq_toolchains")
-use_repo(ext, "yq_toolchains")
\ No newline at end of file
+use_repo(ext, "yq_toolchains")
+register_toolchains("@jq_toolchains//:all")
```

As `register_toolchain` expects a list of labels as an argument, not an individual label, you would have to turn all arguments into singleton lists.
```suggestion
register_toolchains(["@jq_toolchains//:all"])
```

---

Quick question around this topic. I tried to take care of this and I see CI is failing now saying it expects a string rather than a list. e.g. 
```bash
File "aspect_bazel_lib@0.11.0/MODULE.bazel", line 16, column 20, in <toplevel>
--
Error in register_toolchains: at index 0 of register_toolchains, got element of type list, want string
```

I've triple checked the syntax is the same as the example you provided but not 100% sure how to tackle that one yet sorry for the silly question :)

---

Please forget my original comment, you were right: `register_toolchains` takes an arbitrary number of positional arguments of type string, so your initial version was entirely correct.